include/circt/Dialect/HW/Passes.td

```diff
@@ -58,4 +58,14 @@ def VerifyInnerRefNamespace : Pass<"hw-verify-irn"> {
   let constructor =  "circt::hw::createVerifyInnerRefNamespacePass()";
 }
 
+def LowerHWtoBTOR2 : Pass<"hw-lower-to-botr2", "mlir::ModuleOp"> {
```

Comment:
If this produces a btor2 dialect, then it's a lowering, otherwise it's a conversion/emission.